tutorials/08-implementing-a-custom-feature.md

```diff
@@ -175,7 +174,8 @@ implement the `RetrieveWorld` feature function using Dartsim API.
 After defining the feature template, we can add it to a custom
 \ref ignition::physics::FeatureList "FeatureList":
 
-\snippet dartsim/src/CustomFeatures.hh
+[//]: # (TODO: snippet does not render for some reason)
```

Comment:
I think only code in `examples/` can be included like this per the doxygen settings: https://github.com/ignitionrobotics/ign-cmake/blob/3971db877d5aca63e2ff55e31ff8367af14c8a3d/doc/doxygen/api.in#L862